history/characters/1_azerothian.txt

```diff
@@ -17177,4 +17178,109 @@
 		add_trait=lunatic
 	}
 	608.3.10={ death=yes }
+}
+
+#dynasty=413
+#Windsor
+#Lore dynasty of Reginald Windsor
+5820={
+	name=Isiah			#Reginald's greatgrandfather
+	dynasty=413
+	dna="0cebc000ca0"
+	properties="we0jga0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=6 stewardship=3 intrigue=4 learning=4
+	495.10.8 = { birth=yes add_trait=creature_human }
+	522.7.13 = {
+		add_spouse=5830		#Reginald's greatgrandmother
+		effect = { add_artifact_ironfoe_effect = yes }
+	}
+	550.6.13={ death=yes }
+}
+5821={
+	name=Landen			#Reginald's grandfather
+	dynasty=413
+	dna="0lahj0f0c00"
+	properties="we0jga0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=6 stewardship=3 intrigue=4 learning=4
+	father=5820		# Isiah
+	mother=5830		# Dalia
+	523.10.8 = { birth=yes add_trait=creature_human }
+	546.7.13 = {
+		add_spouse=5835		#Reginald's grandmother
+	}
+	550.6.13 = { effect = { add_artifact_ironfoe_effect = yes } }
+	574.6.13={ death=yes }
+}
+5822={
+	name=Scot			#Reginald's father
+	dynasty=413
+	dna="000fe0f0d00"
+	properties="ei0efg0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=5 diplomacy=4 stewardship=5 intrigue=3 learning=4
+	father=5821		# Landen
+	mother=5835		# Ysabel
+	547.3.20 = { birth=yes add_trait=creature_human }
+	569.4.16 = {
+		add_spouse=5840		#Reginald's mother
+	}
+	574.6.13 = { effect = { add_artifact_ironfoe_effect = yes } }
+	590.1.1={ death=yes }
+}
+#Reginald Windsor
+5823={
+	name=Reginald
+	dynasty=413
+	dna="0clif0i0d00"
+	properties="bh0fda0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=6 diplomacy=6 stewardship=4 intrigue=4 learning=4
+	add_trait=tough_soldier add_trait=robust add_trait=stubborn add_trait=honest add_trait=brave
+	father=5822		# Scot
+	mother=5840		# Jane
+	572.1.1 = { birth=yes add_trait=creature_human }
+	588.1.1 = { add_trait=class_warrior_6 }
+	590.1.1 = {
+		employer=8						# Varian Wrynn
```

Comment:
Most likely Reginald should be in Llane's court together with his parents in the first bookmark since...
> He was a loyal friend of Varian Wrynn and studied swordcraft with him

Reply:
done